packages/integrations/aws/README.md

```diff
@@ -0,0 +1,64 @@
+# @astrojs/aws
+
+This adapter allows Astro to deploy your SSR site to [AWS](https://aws.amazon.com/).
```

Comment:
Just a note that we only have docs for static AWS deployments at this link. So, adding this would require a PR to docs, including updating the filtering label etc. and full instructions. (See e.g. Netlify for an example of a deployment option with both static and SSR deploys).

Reply:
Yeah that makes sense. I will comment out this line for now:
> Learn how to deploy your Astro site in our [AWS deployment guide](https://docs.astro.build/en/guides/deploy/aws/).
And if this PR gets accepted, I will submit another PR for the deploy to AWS guide.